### ISSUE TITLE
Chosen language lost at every (non logged-in) return to the website

### ISSUE THREAD

**Issue by nonno** [BeWelcome/rox#206]
I noticed a not perfect behaviour of language selection. When a non logged-in user choose a language, this one is kept till the logout, and re-visiting the homepage, the selected language is English again.
Being an Italian translator I like to have it in Italian since the beginning.
I would change the behaviour like this:
- check the presence of a cookie for the chosen language
- if the cookie is not present, check the `accept-language` array in the request's headers, using the first one available
- when the user manually changes the language with the selector on the top left, that choice is saved in the cookie (with no expiration date)

Verified on both Chrome and Firefox (last versions). Of course it's just a low-priority misbehaviour.

**Comment by thisismeonmounteverest** [comment]
@nonno Is this still the case with the code currently deployed?

**Comment by nonno** [comment]
@thisismeonmounteverest yep. Not only: I'm on the landing page, I set Italian as language (=> Italian), I log-in (=> Italian). If I logout I'm in English again. No big deal but it's not an optimal behaviour.

**Comment by thisismeonmounteverest** [comment]
What language is set in your browser? (I need the locale information.)

Firefox:
![image](https://user-images.githubusercontent.com/11598088/108626379-1a288180-7450-11eb-8e42-7fd64fddd9a5.png)

Chrome:
![image](https://user-images.githubusercontent.com/11598088/108626421-53f98800-7450-11eb-8f8b-a216f317cd53.png)

**Comment by nonno** [comment]
![Screenshot 2021-02-21 at 14 24 51](https://user-images.githubusercontent.com/4105325/108626531-e69a2700-7450-11eb-9c76-70bdc6eda1e0.png)
![Screenshot 2021-02-21 at 14 26 10](https://user-images.githubusercontent.com/4105325/108626533-e7cb5400-7450-11eb-8175-06f077b41a84.png)

**Comment by thisismeonmounteverest** [comment]
@nonno The current behavior I see is that accessing the page at https://beta.bewelcome.org when logged out the page is displayed in Italian as expected. After login it switches to my preferred language and on logout the Language is English. As soon as I refresh it is Italian again.

I try to figure out why it imposed the English on logout.

**Comment by nonno** [comment]
@thisismeonmounteverest for what I can see both in beta and in production at the first opening of the home the page (not logged) is always in English. This even if the request has `it-IT,it;q=0.9,en-US;q=0.8,en;q=0.7,fr-FR;q=0.6,fr;q=0.5,pt-BR;q=0.4,pt-PT;q=0.3,pt;q=0.2` as `Accept-Language`, like in my case. In an ideal world when not logged the language selected should be always the first one available of those in that array. No big deal, but it could be little bit a barrier for someone arriving there for the first time.
